source/class/qx/tool/cli/commands/Test.js

```diff
@@ -56,6 +56,20 @@ qx.Class.define("qx.tool.cli.commands.Test", {
         describe: "Disables the start of the webserver",
         default: false,
         type: "boolean"
+      },
+
+      class: {
```

Comment:
It would be better to use a different word eg testClass because in upcoming release we will support native properties and as class is a reserved word, that will break

Reply:
@johnspackman Does using the argument class in double quotes solve the problem? and the another nuance in docs there is next usage of the command test:

> To see the test results as output of a command line script (in the syntax of the [TAP protocol](https://node-tap.org/tap-protocol/)), execute npx qx test. This will run the server similarly to npx qx serve, and run the tests in a headless browser. To run a single test, you can use qx test --class myapp.test.MyTest --method testOne

If I change names of the arguments there should be changes applied to docs too.

Reply:
Quoting it should be enough, no?

Reply:
That is true ... maybe I'm being pedantic:) I like not having to quote things though, it seems like an unnecessary collision